packages/server/src/arpa_reporter/services/revalidate-uploads.js

```diff
@@ -4,14 +4,12 @@ const { uploadsInPeriod } = require('../db/uploads');
 async function revalidateUploads(period, user, trns) {
     const uploads = await uploadsInPeriod(period.id, trns);
 
-    const updates = [];
-    for (const upload of uploads) {
-        const errors = await validateUpload(upload, user, trns);
-        updates.push({
-            upload,
-            errors,
-        });
-    }
+    const updates = Promise.all(
+        uploads.map(async (upload) => {
+            const errors = await validateUpload(upload);
+            return { upload, errors };
+        }),
+    );
```

Comment:
This edit changes behaviour.

I'm fairly certain I've replicated the existing behaviour but added support for parallelization, but it's worth looking over with another pair of eyes!

Reply:
Overall nice! 

I wonder though, the previous version had three args for `validateUpload`, and the new only has one. You should probably add back `user` at least (though maybe also `trns` for consistency)